commands/command_utils.py

```diff
@@ -165,34 +165,48 @@ def get_target_versions_from_config(src_version_id, distro, flavor):
     return upgrade_paths_map.get(distro, {}).get(flavor, {}).get(src_version_id, [])
 
 
-def get_supported_target_versions(flavour=get_upgrade_flavour()):
+def get_virtual_version_from_config(src_version_id, distro):
+    """
+    Retrieve the virtual version for the given version from upgrade_paths_map.
+
+    :return: The virtual version or None if no match.
+    """
+    upgrade_paths_map = get_upgrade_paths_config()
+    return upgrade_paths_map.get(distro, {}).get('_virtual_versions').get(src_version_id)
```

Comment:
Running on some other distro such as Rocky, this results in an unmanaged crash
```suggestion
    return upgrade_paths_map.get(distro, {}).get('_virtual_versions', {}).get(src_version_id)
```